packages/fixtures/src/adapters/blob/content-loader.ts

```diff
@@ -48,27 +46,15 @@ export class BlobContentLoader implements ContentLoader {
     }));
   }
 
-  async load(
-    messageContents: MessageContentWithIds[],
-  ): Promise<{ error?: Error; success: boolean }> {
-    try {
-      await Promise.all(
-        messageContents.map((content) => {
-          const stringContent = JSON.stringify(content);
-          const blobClient = this.containerClient.getBlockBlobClient(
-            `${content.messageId}.json`,
-          );
-          return blobClient.upload(stringContent, stringContent.length);
-        }),
-      );
-      return { success: true };
-    } catch (error) {
-      return {
-        error: new Error(
-          `Something went wrong trying to load ${messageContents.length} message contents | ${error}`,
-        ),
-        success: false,
-      };
-    }
+  async load(messageContents: MessageContentWithIds[]): Promise<void> {
+    await Promise.all(
```

Comment:
```suggestion
    return Promise.all(
```

Reply:
Sorry I don't get the point, I agree that it should be better to return the promise so the caller can decide weather to wait or not but [here](https://github.com/pagopa/io-messages/pull/184#discussion_r2042394850) we said to return void